src/index.test.ts

```diff
@@ -1059,7 +1059,13 @@ test.each(['__proto__', 'prototype', 'constructor'])(
   forbidden => {
     expect(() => {
       SuperJSON.serialize({
-        [forbidden]: 1,
+        [forbidden]: 1, // doesn't need entry in `meta`, so it's not a risk
```

Comment:
@jonluca I believe this test case should cover your example from https://github.com/blitz-js/superjson/pull/267#issuecomment-1781807466.